data/_ui/tooltips.txt

```diff
@@ -267,18 +267,27 @@ tip "hull:"
 tip "hull energy:"
 	`Energy consumed per second when repairing the hull.`
 
+tip "delayed hull energy:"
+	`Additional energy consumed per second when repairing the hull once no hull damage has been taken for longer than the value of hull delay.`
```

Comment:
I don't really like the second half of the sentence. I'll try to dream up something better, but I'm leaving this comment there for other reviewers/devs until then.

Reply:
I don't like it either, but couldn't think of anything better.

Reply:
```suggestion
	`Additional energy consumed per second to repair the hull once delayed hull repair is active.`
```
Like this?

Reply:
```suggestion
	`Additional energy consumed per second while repairing the hull after the repair delay period has elapsed without hull damage taken.`
```
maybe?

Reply:
For consistency, I'd like to keep the first part as-is (`when repairing`), as that's used in other tooltips. Also, we don't need to fit everything into a single sentence.
```suggestion
	`Additional energy consumed per second when repairing the hull after the repair delay period. The delay restarts every time you take damage.`
```
Still not perfect, though.

Reply:
The "The delay restarts every time you take damage" part would be incorrect, as if you have disabled repair delay but not hull delay, it will only be delayed when you are disabled, not when you take damage. Ditto for shields with depleted delay (such as the ka'het grand restorer).

Reply:
The second part has to go, then.
```suggestion
	`Additional energy consumed per second when repairing the hull after the repair delay period.`
```